### DISCUSSION TITLE
Figma plugin / integration pricing question ...

### DISCUSSION THREAD

**Discussion by stephenhauck** [unoplatform/uno#15678] [Q&A]
I'm not a Figma user so excuse the naiveté ..

In the blog post yesterday there was a mention of the "freemium" model 
https://platform.uno/blog/uno-platform-for-figma-plugin-rtm/

Is this cost above and beyond the cost for Figma itself ?

For example, Figma Professional is $12 a month so is the $9 fee for the Uno plugin / integration on top of the Figma fee for a total of $21 a month ?

Thanks

**Comment by sasakrsmanovic** [comment]
Hello. Yes, the cost  for our plugin is above anything Figma offers. Note that Figma has a free offering too.

**Reply by stephenhauck** [reply]
To clarify .. everything is possible with the FREE version of Figma ?
If not can you be specific about what is not possible since I do not know Figma at all ?
Thanks

**Reply by NVLudwig** [reply]
Hello @stephenhauck,
All features of Uno's plugin work fully with the Free version of Figma. There is a notable Figma feature not available in the FREE version of Figma it's called DevMode. Uno's plugin supports DEV Mode but is not dependant on it in any way so all plugin features work in the Free version of Figma.

Here is a break down of Figma's pricing with the related feature set offered for each plan. 
The uno plugin fully supports all of these.
https://www.figma.com/pricing/
